### ISSUE TITLE
Requesting for feedback-analytics url

### ISSUE THREAD

**Issue by reenamadamwar** [openedx-vlead/vlabs-analytics-api-server#41]
@ksripathi 

For visualization of analytics feedback, I am fetching data from feedback analytics url. Currently this [url](feedback-analytics.vlabs.ac.in/vlabs/feedback/_search?size=10000) throwing 503 Service Temporarily Unavailable error. Without this dashboard will not work.

Check this [issue](https://github.com/vlead/analytics-dashboard/issues/42) for details related about it.

**Comment by ksripathi** [comment]
@reenamadamwar 

I have configured [url](http://analytics-db.base1.vlabs.ac.in/vlabs/feedback/_search?size=10000) which meets your requirement please use this

**Comment by reenamadamwar** [comment]
@ksripathi 

Above mentioned url is having only 2016 year data. Rather, it should have from 2015 till date.

**Comment by reenamadamwar** [comment]
@ksripathi 

url should contain following 6 versions of feedback data from year 2015 till date

1. virtual labs feedback v1.0
2. generic virtual labs feedback v2.0
3. generic labs feedback v2.0
4. generic experiments feedback v2.0
5. Open-Edx labs feedback v2.0
6. Open-Edx experiments feedback v2.0

**Comment by ksripathi** [comment]
@reenamadamwar 

Please use [url](http://analytics-db.base1.vlabs.ac.in/vlabs/feedback/_search?size=10000) which meets your requirement. Let me know if still not found

**Comment by reenamadamwar** [comment]
@ksripathi 

Thank you. Everything is covered in recently provided url.

**Comment by ksripathi** [comment]
@reenamadamwar 

hereafter use attached json file instead making http request. This will reduce your loading cost. I will be sending you file over email

**Comment by reenamadamwar** [comment]
@ksripathi 

Yes I got it thank you. And I am coding in such a way that it will work for url also when we replace it with json file
